test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsUtilityTests.cs

```diff
@@ -316,5 +316,14 @@ public void GetDefaultPushSource_WithNullSettings_Throws()
             ex.Should().NotBeNull();
             ex.Should().BeOfType<ArgumentNullException>();
         }
+
+        [Fact]
+        public void GetForceUpdatePackageLastAccessTimeEnabledStatus_WithNullSettings_Throws()
+        {
+            var ex = Record.Exception(() => SettingsUtility.GetForceUpdatePackageLastAccessTimeEnabledStatus(settings: null));
```

Comment:
Can we can tests for the actual reading of the configuration constant?

Reply:
Ok, I'll probably change and move around this test altogether if we decide to go for the env variable approach though.

Reply:
Done